pkg/tiller/release_install.go

```diff
@@ -172,22 +174,26 @@ func (s *ReleaseServer) performRelease(r *release.Release, req *services.Install
 			Recreate: false,
 			Timeout:  req.Timeout,
 		}
-		s.recordRelease(r, false)
+		if err := s.recordRelease(r, false); err != nil {
+			return res, fmt.Errorf("Failed to record new Release v%d status. Aborting install, no resources were updated", r.Version)
+		}
 		if err := s.ReleaseModule.Update(old, r, updateReq, s.env); err != nil {
 			msg := fmt.Sprintf("Release replace %q failed: %s", r.Name, err)
 			s.Log("warning: %s", msg)
 			old.Info.Status.Code = release.Status_SUPERSEDED
 			r.Info.Status.Code = release.Status_FAILED
 			r.Info.Description = msg
-			s.recordRelease(old, true)
+			s.recordRelease(old, true) //ignore store errors as can't do much about them anyway
```

Comment:
This is a good opportunity to teach others reading through this piece of code. Can you elaborate more on this comment so others can learn? Why can't tiller do much with these store errors?